### PR TITLE
signup.html: Minor Usability improvements

### PR DESCRIPTION
**Pull request by jdittrich**
* Birth Year as integer (+placeholder) (Out of habit I gave it a date (did not work) and then an ISO-8601 (which worked)) 
* Mail as appropriate type to call up correct soft keyboard on phones.

### CODE REVIEW

**Comment by fpereiro** [comment]
Hi @jdittrich ! This looks great. We apply `parseInt` to the input in client side js, so that in case an ISO date is entered we only keep the year. But it's better to catch the error earlier.

@Felienne do you think that adding "e.g." would be confusing for kids? Perhaps we can simply put a year as placeholder.

File: templates/signup.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h2>{{auth.create_account}}</h2>
     <input id="password_repeat" type=password>
     <br><br>
     <label>{{auth.birth_year}}</label>
-    <input id="birth_year">
+    <input id="birth_year" placeholder="e.g. 1995" type="number">
```
Comment (fpereiro) [inline templates/signup.html:21]: Maybe we can omit the "e.g" in case it confuses some kids?

**Review by fpereiro — APPROVED** [review]

**Comment by Felienne** [comment]
> do you think that adding "e.g." would be confusing for kids? Perhaps we can simply put a year as placeholder.

Year I think e.g. is hard. An example placeholder would be better

**Comment by jdittrich** [comment]
deleted the e.g.

**Comment by fpereiro** [comment]
Merged. Thanks again @jdittrich !

**Pull request merged**